### PR TITLE
isLinkCurrent() arguments

### PR DESCRIPTION
**Pull request by krausv**

### CODE REVIEW

File: latte/cs/macros.texy [hunk 1]
```diff
@@ -419,6 +419,12 @@ Anebo potřebujete označit aktivní menu třídou `current`?
 
 Úžasné, že?
 
+.[caution]
```
Comment (dg) [inline latte/cs/macros.texy]: I think that [caution] is not necessary.

**Comment by krausv** [comment]
ok :)

File: latte/cs/macros.texy (hunk 1 above)
Comment (krausv) [inline latte/cs/macros.texy]: ok :)

**Comment by dg** [comment]
:+1:

**Pull request merged**